gokart/mypy.py

```diff
@@ -329,6 +330,10 @@ def collect_attributes(self) -> Optional[list[TaskOnKartAttribute]]:
             with state.strict_optional_set(self._api.options.strict_optional):
                 init_type = self._infer_task_on_kart_attr_init_type(sym, stmt)
 
+            # infer Parameter type
+            if init_type is None:
+                init_type = self._generate_type_from_parameters(stmt.rvalue)
```

Comment:
please use consistent naming
```suggestion
                init_type = self._infer_type_from_parameters(stmt.rvalue)
```